lessons/2_more_strings_and_functions/lesson.py

```diff
@@ -199,7 +231,10 @@ def inner_func():
 # Please list the globally available names in this list, with comman separation and quotes
 # i.e. like this STUFF = ["FIRST THING", "SECOND THING", "ETC"]
 
-GLOBAL_NAMES = []
+GLOBAL_NAMES = ["var1", "cool_func"]
+# # cool_func added after failing test.py.
+# # SHOULD HAVE BEEN RUNNING THE TEST, because until this moment i had not realized that
```

Comment:
Functions don't define a variable, but they do define a name. You could also set cool_func equal to something that isn't a function and it would overwrite the cool_func definition if you reset it tosomething at the global scope level. So it isn't that cool_func is a variable, it's that the name cool_func is defined to mean something, which happens to be a function, and var1 is defined to mean something, which happens to be a variable. The code could be rewritten this way:

```
cool_func = "JJ Flewellen"

def var1():
    var2 = "CAT"
    def inner_func():
        print("I am the inner")
    return "cool"
```

And the answer to GLOBAL_NAMES would be the exact same.